packages/dev/inspector/src/components/sceneExplorer/entities/gui/guiTools.ts

```diff
@@ -45,8 +45,9 @@ export function SetGUIEditorURL(guiEditorURL: string) {
  * if you are in an ES6 environment, you must first call InjectGUIEditor to provide the gui-editor package
  * If you are in a UMD environment, it will load the package from a URL
  * @param adt
+ * @param fromPG defines whether editor is being opened from the Playground
  */
-export async function EditAdvancedDynamicTexture(adt: AdvancedDynamicTexture) {
+export async function EditAdvancedDynamicTexture(adt: AdvancedDynamicTexture, fromPG?: boolean) {

```

Comment:
maybe instead of `fromPG` we can use `embed` or something like that?

Reply:
Is that change due to just having better naming conventions, or because I've also defined fromPG in globalState? I'm just wondering if I should change fromPG in globalState to embed as well?